source/mainnet/net/installation/downloads.rst

```diff
@@ -59,33 +87,63 @@ The |bw| is available to download as an extension for the `chromium web browsers
+         - :ref:`Verification instructions <verification-cdw-deb>`
+
+      -  `Mainnet RPM <https://distribution.mainnet.concordium.software/tools/linux/concordium-desktop-wallet-1.5.0.rpm>`_
+
+         - SHA256 checksum of the download: :substitution-code:`|cdw-rpm-checksum|`
+         - :ref:`Verification instructions <verification-cdw-rpm>`
+
+.. _dw-testnet-download:
+
+.. dropdown:: Testnet
+
+   .. dropdown:: Windows v1.5.0
 
--  `Download the Desktop Wallet for Windows <https://distribution.mainnet.concordium.software/tools/windows/concordium-desktop-wallet-1.5.0.exe>`_
+      `Download the Testnet version of Concordium Desktop Wallet for Windows <https://distribution.testnet.concordium.com/tools/windows/concordium-desktop-wallet-testnet-1.5.0.exe>`_
 
-MacOS v1.5.0
-------------
--  `Download the Desktop Wallet for macOS <https://distribution.mainnet.concordium.software/tools/macos/concordium-desktop-wallet-1.5.0.dmg>`_
+   .. dropdown:: MacOS v1.5.0
 
-Linux® v1.5.0
--------------
--  Download the Desktop Wallet for Linux®:
+      `Download the Testnet version of Concordium Desktop Wallet for MacOS <https://distribution.testnet.concordium.com/tools/macos/concordium-desktop-wallet-testnet-1.5.0.dmg>`_
 
-   -  `Mainnet AppImage <https://distribution.mainnet.concordium.software/tools/linux/concordium-desktop-wallet-1.5.0.AppImage>`_
+   .. dropdown:: Linux® v1.5.0
 
-      - SHA256 checksum of the download: :substitution-code:`|cdw-appimage-checksum|`
-      - :ref:`Verification instructions <verification-cdw-appimage>`
+      Download the Testnet version of Concordium Desktop Wallet for Linux®:
 
-   -  `Mainnet Debian package <https://distribution.mainnet.concordium.software/tools/linux/concordium-desktop-wallet-1.5.0.deb>`_
+      -  `Testnet AppImage <https://distribution.testnet.concordium.com/tools/linux/concordium-desktop-wallet-testnet-1.5.0.AppImage>`_
+         -  SHA256 checksum of the download: ``044d8b2aa039428b7e6626d3af1dade21e52bee04dbcfc9f78d303bd0f1c855e``
 
-      - SHA256 checksum of the download: :substitution-code:`|cdw-deb-checksum|`
-      - :ref:`Verification instructions <verification-cdw-deb>`
+      -  `Testnet Debian package <https://distribution.testnet.concordium.com/tools/linux/concordium-desktop-wallet-testnet-1.5.0.deb>`_
+         -  SHA256 checksum of the download: ``f15d996f9e73118cf12820265af6303a9d6359b30c1e3c401bd0f292e977ceb2``
 
-   -  `Mainnet RPM <https://distribution.mainnet.concordium.software/tools/linux/concordium-desktop-wallet-1.5.0.rpm>`_
+      -  `Testnet RPM <https://distribution.testnet.concordium.com/tools/linux/concordium-desktop-wallet-testnet-1.5.0.rpm>`_
+         -  SHA256 checksum of the download: ``66d2a9ce4809a6e84a84accdb0768d6c1c6a7dcb046b751667a80d75a8a66464``
 
-      - SHA256 checksum of the download: :substitution-code:`|cdw-rpm-checksum|`
-      - :ref:`Verification instructions <verification-cdw-rpm>`
+.. _ledger-app-download:
 
 Concordium LEDGER App
```

Comment:
This also hasn't changed in this PR specifically, but if people haven't read anything about our LEDGER app, and then just checks out what there is to download, the public key comment for the certificate doesn't really make sense: 

![image](https://github.com/Concordium/concordium.github.io/assets/76045015/036bad42-d2a4-4962-8aec-44a00f987504)

Reply:
I don't understand your comment. Can you explain?

Reply:
I can try! 

If people don't know about the side loading, they might not understand what "When installing the certificate ..." means. The sentence sort of just comes out of the blue.

Reply:
I moved it to below the downloads. Martin asked specifically for this to be included a few months ago.